visualization/motion_planning_tasks/src/task_list_model.cpp

```diff
@@ -291,6 +291,7 @@ void TaskListModel::processTaskStatisticsMessage(const std::string& id,
                                                  const moveit_task_constructor_msgs::TaskStatistics& msg) {
 	auto it = remote_tasks_.find(id);
 	if (it == remote_tasks_.cend()) {
+		// TODO(v4hn): cache as we might receive this before the actual description
```

Comment:
Yeah, probably we should handle this case... Have you actually seen this warning already?